### DISCUSSION TITLE
Use my fork of EeveeSpotify

### DISCUSSION THREAD

**Discussion by yodaluca23** [SpotCompiled/SpotC-Plus-Plus#45] [Polls]
Would anyone be opposed to me switching to using my fork of EeveeSpotify? It has an extra (really good) Lyric Source, and it might allow me to allow the update popup, to link to this repo...

**Comment by yodaluca23** [comment]
https://github.com/yodaluca23/EeveeSpotify

**Reply by Ilovecatz17** [reply]
What's the difference between your eveespotify and the original??

**Reply by yodaluca23** [reply]
Read the original post?

**Reply by Ilovecatz17** [reply]
I did, but im still confused. how can the lyric source be better than the other one? musixmatch is great. does yours have time sync with more songs?

**Reply by yodaluca23** [reply]
It uses a different API... It uses the API for the BeautifulLyrics Spictify extension, which itself gets lyrics from several different providers, it has time synced lyrics for basically any song you can think of...

**Reply by Ilovecatz17** [reply]
oh wow. thats actually amazing. are the lyrics the only difference?

**Reply by yodaluca23** [reply]
> oh wow. thats actually amazing. are the lyrics the only difference?

For now yes, but I'm working on making the update notification function, not look at the EeveeSpotify repo, but rather this repository so users don't get confused and try to download the original EeveeSpotify tweak.

**Comment by Blackbackofficial** [comment]
I would prefer to keep both options. We could make a switcher in the EeveeSpotify application settings to toggle between your version and the original version of EeveeSpotify, thus allowing users to decide which one they like more.

How complicated is the implementation of this idea?

**Reply by yodaluca23** [reply]
> I would prefer to keep both options. We could make a switcher in the EeveeSpotify application settings to toggle between your version and the original version of EeveeSpotify, thus allowing users to decide which one they like more.

The source I added is just like any other from the dropdown you don't have to use it, it's just another option...

> How complicated is the implementation of this idea?

Very

**Reply by Blackbackofficial** [reply]
Okey-dokey, I wanna see your option

**Comment by SourcedX777** [comment]
What other features are you considering to add?

**Reply by yodaluca23** [reply]
Changing the Update notification to check for updates from this repo, so people don't get confused and try to download the original EeveeSpotify Tweak.
